src/databases/Xolotl/CMakeLists.txt

```diff
@@ -54,7 +54,9 @@ ${VISIT_INCLUDE_DIR}/visit_vtk/lightweight
 ${VTK_INCLUDE_DIRS}
 )
 
-LINK_DIRECTORIES(${VISIT_LIBRARY_DIR} ${HDF5_LIBRARY_DIR} )
+LINK_DIRECTORIES(${VISIT_LIBRARY_DIR} ${HDF5_LIBRARY_DIR}
+  ${ZLIB_LIBRARY_DIR}
```

Comment:
ZLIB_LIBRARY_DIR isn't necessary in the link_directories. Instead change ZLIB_LIB in the target_link_libraries call to ZLIB_LIBRARY which contains full path.

Reply:
Done.